ydb/tests/compatibility/test_vector_index.py

```diff
@@ -0,0 +1,212 @@
+import pytest
+import time
+import random
+from ydb.tests.library.compatibility.fixtures import RestartToAnotherVersionFixture
+from ydb.tests.oss.ydb_sdk_import import ydb
+
+TABLE_NAME = "table"
+
+
+class TestVectorIndex(RestartToAnotherVersionFixture):
+    @pytest.fixture(autouse=True, scope="function")
+    def setup(self):
+        self.table_name = TABLE_NAME
+        self.rows_count = 15
+        self.index_name = "vector_idx"
+        self.vector_dimension = 5
+        yield from self.setup_cluster(extra_feature_flags={"enable_vector_index": True})
+
+    def _get_random_vector(self, type, size):
```

Comment:
The parameter name type shadows the built-in; consider renaming it to vector_type for clarity.